### ISSUE TITLE
Inputtext listener interface how to use ?

### ISSUE THREAD

**Issue by yasinkasap** [TheCoder4eu/BootsFaces-OSP#818]
Hi,

I want to use for keep value changes on autocomplete input text component. At showcase page, in the component's attribute list value-change-listener exist but i didn't find listener interface info. I'm using tree component alsa and i used the component's listener interface : TreeNodeEventListener. At the same way, how can i use Inputtext's value change listener interface class.

Thanks, regards.

**Comment by stephanrauh** [comment]
That's several questions in a single ticket, isn't it?

- value-change-listener: That's just the standard JSF valueChangeListener.
- b:inputText changeListener: I guess that's also a standard JSF listener.

But: We've implemented the standard JSF AJAX attributes as an afterthought. We didn't want to do that, and we still support them reluctantly. We've only added support for the standard API by public demand. The BootsFaces way to do it is using the JavaScript events and add the "ajax:" prefix to it. Refer to https://showcase.bootsfaces.net/forms/ajax.jsf to read the full story.
